src/Microsoft.Performance.SDK.Runtime/ExecutionContext.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Performance.SDK.Runtime
 {
     /// <summary>
     ///     Represents the context of execution for
-    ///     a custom data source.
+    ///     an <see cref="IProcessingSource"/>.
     /// </summary>
     public sealed class ExecutionContext
```

Comment:
>ExecutionContext [](http://example.com/codeflow?start=24&length=16)

should we rename this to ProcessingContext for consistency?

Reply:
This class is also associated more with ProcessingSourceExecutor than ProcessingSource